packages/shorebird_cli/lib/src/shorebird_flutter.dart

```diff
@@ -218,4 +218,31 @@ class ShorebirdFlutter {
     shorebirdEnv.flutterRevision = revision;
     useFlutterProgress.complete();
   }
+
+  /// Returns an human readable string from the received [flutterRevision] and
+  /// [flutterVersion].
+  ///
+  /// If [flutterRevision] is null, the current revision will be used.
+  /// If [flutterVersion] is null, it will be calculated from the current
+  ///   revision using [getVersionString].
+  Future<String> humanReadableVersion({
```

Comment:
Why does this accept a flutterRevision and flutterVersion instead of using the environment?

Reply:
I've changed for it to require the `flutterRevision`, so it will never take from the env.

The flutter version is optional because the user of this function might already have that value available (like when dealing with an `AppRelease`, so it can be passed and when that is done, this method will not try to "calculate" the version from the revision.

Reply:
The API is pretty confusing imo. If we already have the version and the revision we shouldn't need to call this method at all right?

Reply:
I think this is all a sign that we probably have the wrong abstraction. The different methods we need are:

1. Get the Flutter Version
2. Get the Flutter Revision
3. Format the Flutter Version and Flutter Revision into a human-readable string.

Right now this one method is attempting to do too much imo.